server/models/user.py

```diff
@@ -8,11 +8,15 @@ class User(db.Model):
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     email = db.Column(db.String(255), unique=True, nullable=False)
+    name = db.Column(db.String(255), nullable=False)  # Right now this is a very simple full name column. Could be
+    # possibly better to include first and last name fields in the front-end form and make this two columns, one for
+    # first names and one for last names.
```

Comment:
Maybe. On the other hand, this way gives the user the freedom to use whatever they wish for their name, if they like to go by one name or 3+. I think it's fine for our use case.